### PR TITLE
Proof of concept for publishing job offers in website

### PR DESCRIPTION
**Pull request by dukebody**
Just a proof of concept to see how easy or hard would this be. Tell me what you think!

### CODE REVIEW

File: content/job-offers/bmat-full-stack-developer-backend/contents.lr [hunk 1]
```diff
@@ -0,0 +1,69 @@
+pub_date: 2018-05-19
+---
+description:
+
+
+BMAT gathers a group of young talented professionals with an extraordinary passion for music and technology. We aim to make music a better place for artists, agents and users, and we are constantly coming up with new ideas of how to develop our systems and services.
```
Comment (rberenguel) [inline content/job-offers/bmat-full-stack-developer-backend/contents.lr]: I'd wrap long lines, but aside from that it looks straightforward enough

Comment (rberenguel) [reply content/job-offers/bmat-full-stack-developer-backend/contents.lr]: We have soft wrapping in our editors, but when checking the PR in github, we don't have it. So it looks weird and would make quick PR merging more of a pain here.

Comment (ifosch) [reply content/job-offers/bmat-full-stack-developer-backend/contents.lr]: In this specific topic, I don't like to scroll horizontally to read lines. As I said previously, I would prefer to have lines split on periods, at best.

**Comment by ifosch** [comment]
I think this is overkill for the job offer publishing.
Publishing job offers is already a pretty laborious work IMO: Manage the validation, write the email, write the tweet, reply the proposal... And, again IMO, this happens too often to have a new post being published.
I thought this task was supposed to make the whole process more automated, not more manual work.
Maybe we should discuss this in depth, since I think we could be using FaaS or similar to handle this, instead.

**Comment by DZPM** [comment]
+1 to ifosch, we should go to a more automatic approach.

With this PR, we would have to re-write and re-format every mail.

**Comment by dukebody** [comment]
Hi!

I've submitted in this branch a small script to render and send job offers via email, taking them from the website. It can be used to automate sending the email from travis-ci, for example, checking if a new offer has been added under the `job-offers` folder in git. It can also be used from a developer machine, I've tested it myself with one of the job offers in the branch.

It is of course more work than just copy pasting and re-formatting the received email, since it has to be formatted in Markdown, although there are tools to assist with this (stackedit.io), create a commit, do a PR, etc.. However IMHO this is the bad side of using static site generators: you don't have a WYSIWYG editor where you can copy-paste and re-format easily, or plugins to automatically send tweets and mails when you publish a new post that you can setup with almost no work, etc. Simplicity comes with a cost.

I still see value in having the offer published in the website, to make it searchable at least. But I don't know if this added value is enough to justify spending a lot of time in this, since people looking for a job can always get to https://www.meetup.com/python-185/messages/archive/ to see the job offers, and this might be enough.

We can spend efforts instead in writing "blog" posts about news, upcoming events, etc. :)

**Pull request closed without merging**